manager-rs/dauth-service/src/rpc/handlers/backup_network.rs

```diff
@@ -358,19 +371,22 @@ impl BackupNetworkHandler {
                 .await?;
 
                 // collect all properly formated delegated shares
-                // skip errors
+                // log and skip on error
+                let mut processed_shares = Vec::new();
```

Comment:
As called out on the client side of this, the actual share generation will need to be a little more complex, since the share dissemination is to multiple backup networks.